server/types/Types.ts

```diff
@@ -0,0 +1,62 @@
+export type SpanCache = {
+    processNum:string,
+    spanIds: Array<string>
+    spanData: Array<string>
+  };
+
+export type EdgeData = {
+    data: {
+        source: string,
+        target: string,
+        type: 'arrow',
+        label: any,
+    },
+        classes: 'background'
+   };
```

Comment:
This looks poorly linted. We need to install eslint to get things cleaned up at some point. Let's add a task for it?